cmd/sourced/compose/workdir/workdir.go

```diff
@@ -27,14 +27,22 @@ var (
 	ErrMalformed = goerrors.NewKind("workdir %s is not valid: %s")
 )
 
+// WorkdirType defines the type of the workdir
 type WorkdirType int
 
 const (
+	// None refers to a failure in identifying the type of the workdir
 	None WorkdirType = iota
+	// Local refers to a workdir that has been initialized for local repos
 	Local
+	// Orgs refers to a workdir that has been initialized for organizations
 	Orgs
 )
 
+// Workdir represents the workdir:
+//   - Type: is the WorkdirType
```

Comment:
you can document fields of the struct instead.

Reply:
Addressed [here](https://github.com/src-d/sourced-ce/pull/183/commits/176d2e905f8d1cca2913acb2dfd60a93eb6daa2a).